lib/components/artist/shows/index.js

```diff
@@ -47,7 +47,9 @@ export default Relay.createContainer(Shows, {
   fragments: {
     artist: () => Relay.QL`
       fragment on Artist {
-        ${CurrentShows.getFragment('artist')}
+        current_shows: partner_shows(status: "running") {
```

Comment:
This uses a [GraphQL alias](https://facebook.github.io/graphql/#sec-Field-Alias) to give the resulting collection a name. Otherwise it would just be called after the property (`partner_shows`) which would clash when adding the queries for `upcoming` and `past` shows.